examples/plots/plot_interpolation_for_transform_manager.py

```diff
@@ -0,0 +1,101 @@
+"""
+======================
+Managing Transformations over Time
+======================
+
+In this example, given two tranformation trajectories, we will interpolate both 
+and use the transform manager for the target timestep.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from pytransform3d import rotations as pr
+from pytransform3d import transformations as pt
+from pytransform3d.transform_manager import TransformManager
+
+
+def create_sinusoidal_movement(duration_sec, sample_period, x_velocity, y_start_offset, start_time):
+    """Create a planar (z=0) sinusoidal movement around x-axis.
+    """
+
+    time_arr = np.arange(0, duration_sec, sample_period) + start_time
+    N = len(time_arr)
+    x_arr = np.linspace(0, x_velocity*duration_sec, N) 
+
+    spatial_freq = 1/5  # 1 sinus per 5m
+    y_arr = np.sin(2*np.pi*spatial_freq*x_arr)
+    y_arr += y_start_offset
+    
+    dydx_arr = np.cos(2*np.pi*spatial_freq*x_arr)
+    yaw_arr = np.arctan2(dydx_arr, np.ones_like(dydx_arr))
+    
+    pq_arr = list()
+    for i in range(N):
+        R = pr.active_matrix_from_extrinsic_euler_zyx([yaw_arr[i], 0, 0])
+        T = pt.transform_from(R, [x_arr[i], y_arr[i], 0])
+        pq = pt.pq_from_transform(T)
+        pq_arr.append(pq)
+
+    return time_arr, np.array(pq_arr)
+
+
+# create entities A and B
+duration = 10.0  # [s]
+sample_period = 0.5  # [s]
+velocity_x = 1  # [m/s]
+time_W2A, pq_arr_W2A = create_sinusoidal_movement(duration, sample_period,
```

Comment:
Please use pep8 or flake8 to check the coding style: line length and indentation.

Reply:
Fixed with black